### DISCUSSION TITLE
How to use subscriber login?

### DISCUSSION THREAD

**Discussion by fu8765** [heliomarpm/udemy-downloader-gui#109] [Q&A] [answered]
testing the subscriber account login, and i'm not sure whether i'm doing something wrong, or something my organization uses.
1) I click on subscriber account then get credentials
2) A window pops up to login, where I enter my organizations email address, it says that I will receive an email with a link to login
3) go back to my email on my browser and click the link, and I'm logged into the browser but not udemy downloader.

keep in mind, the "receiving link" to login has always been the case for me, whether using udemy downloader or not. and udemy's "account settings" does not seem to have any way to change this behaviour.

Maybe addint a url bar to the pop up browser window, might be a good workaround?

**Comment by heliomarpm** [comment]


**Reply by heliomarpm** [reply, marked as answer]
> Hello, sorry for the delay. Come on, is your account business or personal? If your account is business, you need to enter the company name in the subdomain field. If you use a personal account, you also have the option of informing whether or not you are a subscriber.
> 
> After that, when clicking on login, the window that opens is that of Udemy itself, just follow the same steps as if you were on the Udemy platform

Hello, sorry for the delay.
Come on, is your account business or personal?
If your account is business, you need to enter the company name in the subdomain field.
If you use a personal account, you also have the option of informing whether or not you are a subscriber.

After that, when clicking on login, the window that opens is that of Udemy itself, just follow the same steps as if you were on the Udemy platform
